tcms/testplans/templates/testplans/get.html

```diff
@@ -110,26 +112,27 @@ <h2 class="card-pf-title">
 <template id="test_case_row">
     <div class="list-group-item">
         <div class="list-group-item-header">
-            <div class="list-view-pf-expand">
-                <span class="fa fa-angle-right"></span>
-            </div>
-            <div class="list-view-pf-checkbox">
-                <input type="checkbox">
-            </div>
-            <div class="list-view-pf-actions">
-                <div class="dropdown pull-right dropdown-kebab-pf">
-                    <button class="btn btn-link dropdown-toggle" type="button" id="dropdownKebabRight9"
-                            data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
-                        <span class="fa fa-ellipsis-v"></span>
-                    </button>
-                    <ul class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownKebabRight9">
-                        <li><a href="#">{% trans 'Change status' %}</a></li>
-                    </ul>
-                </div>
-            </div>
             <div class="list-view-pf-main-info">
-                <!-- todo: this text is slightly misaligned. for some reason goes to the top -->
-                <a class="testcase" href="#"></a>
+                <div class="list-view-pf-left">
+                    <span class="fa fa-angle-right"></span>
+                </div>
+                <div class="list-view-pf-body">
+                    <div class="list-view-pf-checkbox">
+                        <input type="checkbox">
+                    </div>
```

Comment:
I think the checkbox div can safely live outside the pf-body section. Doesn't seem to make any difference for me.